app/services/reporting/account_reuse_report.rb

```diff
@@ -173,6 +178,48 @@ def update_from_results(results:, total_registered:, total_proofed:)
             end
           end
         end
+
+        # rubocop:disable Metrics/BlockLength
+        results.each_with_index do |details_section, section_index|
+          details_section.each_with_index do |detail_row, row_index|
+            if detail_row.num_entities >= 10
+              captured_row = detail_row
+              previous_row = details_section[row_index - 1]
+
+              if !previous_row.num_entities.is_a?(String) && previous_row.num_entities < 10
+                new_num_entities = captured_row.num_entities.to_s
+                running_all_users = captured_row.num_all_users
+                running_all_percent = captured_row.all_percent
+                running_idv_users = captured_row.num_idv_users
+                running_idv_percent = captured_row.idv_percent
+                summary_row_index = row_index
+              else
+                new_num_entities = "10-#{captured_row.num_entities}"
+                running_all_users = previous_row.num_all_users + captured_row.num_all_users
+                running_all_percent = previous_row.all_percent + captured_row.all_percent
+                running_idv_users = previous_row.num_idv_users + captured_row.num_idv_users
+                running_idv_percent = previous_row.idv_percent + captured_row.idv_percent
+                summary_row_index = row_index - 1
+
+                details_section.delete_at(row_index)
```

Comment:
just to check how this works, does these iterate and delete one row at a time?

Another approach might be:

```
results.select { |details| details.num_entities > 10 }
```
and if we defined a `#merge` method that adds them together, we could do
```
results.select { |details| details.num_entities > 10 }.reduce(&:merge)
```

Reply:
Okay - cool, I can look at a merge method.

(Yes, it is iterating over the rows and grabbing info and adding it to a single row and deleting the original row.)

Reply:
I didn't create a separate merge method, but I was able to avoid the extended, nested iteration...